source/wp-content/themes/wporg-developer-2023/patterns/single.php

```diff
@@ -7,27 +7,40 @@
 
 ?>
 
-<!-- wp:group {"tagName":"main","style":{"spacing":{"blockGap":"0px","padding":{"top":"var:preset|spacing|50"}}},"className":"entry-content","layout":{"type":"constrained"}} -->
-<main class="wp-block-group entry-content" style="padding-top:var(--wp--preset--spacing--50);">
-
-<!-- wp:wporg/code-reference-deprecated {"style":{"spacing":{"margin":{"bottom":"var:preset|spacing|20"}}}} /-->
-<!-- wp:wporg/code-reference-private-access {"style":{"spacing":{"margin":{"bottom":"var:preset|spacing|20"}}}} /-->
-<!-- wp:wporg/code-reference-title /-->
-<!-- wp:wporg/code-reference-summary /-->
-<!-- wp:wporg/code-reference-description /-->
-<!-- wp:wporg/code-reference-parameters /-->
-<!-- wp:wporg/code-reference-return-value /-->
-<!-- wp:wporg/code-reference-explanation /-->
-<!-- wp:wporg/code-reference-source /-->
-<!-- wp:wporg/code-reference-hooks /-->
-<!-- wp:wporg/code-reference-related /-->
-<!-- wp:wporg/code-reference-changelog /-->
-<!-- wp:wporg/code-reference-comments /--></main>
-
-<!-- wp:group {"tagName":"div","style":{"spacing":{"padding":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|40", "left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space"}}}} -->
-<div class="wp-block-group alignfull" style="padding-top:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--40);padding-left:var(--wp--preset--spacing--edge-space);padding-right:var(--wp--preset--spacing--edge-space)">
-	<!-- wp:wporg/code-reference-comment-form /-->
-</div>
-<!-- /wp:group -->
-<!-- /wp:group -->
+<!-- wp:group {"tagName":"main","layout":{"type":"constrained"},"style":{"spacing":{"padding":{"left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space"}}}} -->
```

Comment:
I'll probably reverse these changes and rebase once we agree on the approach: https://github.com/WordPress/wporg-developer/pull/194#issuecomment-1414521184